management-app/src/main/java/com/sdevelopment/skt/management/ServletInitializer.java

```diff
@@ -0,0 +1,13 @@
+package com.sdevelopment.skt.management;
+
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.support.SpringBootServletInitializer;
+
+public class ServletInitializer extends SpringBootServletInitializer {
```

Comment:
what is the purpose of this class?

Reply:
At the beginning it was generated with the project then I was trying to fix an issue and the possible solution was to extend it directly in the ManagementAppAplication class. I did the changed but kept this Initializer.

Reply:
Right now I'm not sure if is mandatory for the app to run because it was generated by the starter. I will look into it.